packages/@aws-cdk/cloudformation-include/test/test-templates/yaml/short-form-sample.yaml

```diff
@@ -0,0 +1,62 @@
+Resources:
+  RecordServiceS3Bucket2:
+    Type: AWS::S3::Bucket
+    DeletionPolicy: Retain
+    Properties:
+      ReplicationConfiguration:
+        Role: !GetAtt [ WorkItemBucketBackupRole, !Ref 'AWS::Region' ]
+        Rules:
+        - Destination:
+            Bucket: !Join ['', ['arn:aws:s3:::', !Join ['-', [!Ref 'AWS::Region', !Ref 'AWS::StackName',
+                    replicationbucket]]]]
+            StorageClass: STANDARD
+          Id: Backup
+          Prefix: ''
+          Status: Enabled
+      VersioningConfiguration:
+        Status: Enabled
+  RecordServiceS3Bucket:
+    Type: AWS::S3::Bucket
+    DeletionPolicy: Retain
+    Properties:
+      ReplicationConfiguration:
+        Role: !GetAtt WorkItemBucketBackupRole.!Ref 'AWS::Region'
```

Comment:
Can you make sure this syntax is unique to `Fn::GetAtt`, and can't be used with other functions as well?